viewpump/src/main/java/io/github/inflationx/viewpump/internal/-ViewPumpLayoutInflater.kt

```diff
@@ -14,32 +14,21 @@ import org.xmlpull.v1.XmlPullParser
 import java.lang.reflect.Field
 
 @Suppress("ClassName")
-internal class `-ViewPumpLayoutInflater` : LayoutInflater, `-ViewPumpActivityFactory` {
+internal class `-ViewPumpLayoutInflater`(
+    original: LayoutInflater,
+    newContext: Context,
+    cloned: Boolean
+) : LayoutInflater(original, newContext), `-ViewPumpActivityFactory` {
```

Comment:
-ViewPump what's the - for?

Reply:
It's a clever trick that hides it from Java. Classes with leading chars that aren't valid in Java (but are valid in the JVM) just won't index if someone tries to use it

Reply:
Oh and still valid name for the AndroidVM?

Reply:
Yep! Okio does the same approach